src/@next/Tooltip/Tooltip.tsx

```diff
@@ -247,16 +250,25 @@ export const Tooltip = ({
       content
     );
 
-  const handleMouseEnter = () => setIsActive(true);
+  const handleMouseEnter = () => {
+    if (!clickable) setIsActive(true);
+  };
   const handleMouseLeave = () => {
-    setIsActive(false);
-    handleClick();
+    if (!clickable) {
+      setIsActive(false);
+      handleAnimation();
```

Comment:
you won't be needing this after moving the animations to the styles

Reply:
hmm I need to set animate state to true indicating it's time for fade out effect... the animation is handle on the styles, this is just to update the state (whether to animate fade out or not)...